channeldb/channel.go

```diff
@@ -2068,11 +2068,25 @@ type HTLC struct {
 	// from the HtlcIndex as this will be incremented for each new log
 	// update added.
 	LogIndex uint64
+
+	// ExtraData contains any additional information that was transmitted
+	// with the HTLC via TLVs. This data *must* already be encoded as a
+	// TLV stream, and may be empty.
+	ExtraData []byte
```

Comment:
Will there be a size limit? Implicitly, since it's encoded together with onion blob, the size is capped at `66000-1366` or an error is returned,
https://github.com/lightningnetwork/lnd/blob/bbbf7d33fb1527acebb44e2a69d16fbcf24cc2fa/channeldb/codec.go#L406-L412

Reply:
Assuming that this data comes from `UpdateAddHTLC`'s TLV stream (which it should), the size is already limited by protocol message size.